python/ray/data/tests/test_dataset_tfrecords.py

```diff
@@ -213,6 +214,16 @@ def test_write_invalid_tfrecords(ray_start_regular_shared, tmp_path):
         ds.write_tfrecords(tmp_path)
 
 
+def test_read_invalid_tfrecords(ray_start_regular_shared, tmp_path):
+    file_path = os.path.join(tmp_path, "file.json")
+    with open(file_path, "w") as file:
+        json.dump({"number": 0, "string": "foo"}, file)
+
+    # Expect exception raised when reading JSON as TFRecord file.
+    with pytest.raises(Exception, match="Failed to read TFRecord file"):
```

Comment:
If the above exception type is changed.
```suggestion
    with pytest.raises(RuntimeError, match="Failed to read TFRecord file"):
```

Reply:
Updated.